specification/reservations/resource-manager/Microsoft.Capacity/preview/2020-10-01-preview/reservations.json

```diff
@@ -322,6 +322,49 @@
         }
       }
     },
+    "/providers/Microsoft.Capacity/reservationOrders/{reservationOrderId}/changeDirectory": {
```

Comment:
how do you validate that the current identity/caller of this api is authorized to make this change? you check the caller's permission on the destination tenant?

Reply:
the validation happens in the backend. Error response is thrown if the caller does not have permission in the destination tenant